src/Resources/skeleton/crud/templates/index.tpl.php

```diff
@@ -0,0 +1,31 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <h1><?= $entity_class_name; ?> index</h1>
+    <table>
+        <thead>
+            <tr>
+                <?php foreach ($entity_fields as $field): ?><th><?= ucfirst($field['fieldName']); ?></th>
+                <?php endforeach; ?>
+                <th>actions</th>
+            </tr>
+        </thead>
+        <tbody>
+        {% for <?= $entity_var_singular; ?> in <?= $entity_var_plural; ?> %}
+            <tr>
+                <?php foreach ($entity_fields as $field): ?><td>{{ <?= $entity_var_singular; ?>.<?= $field['fieldName'].(false === strpos($field['type'], 'date') ? '' : '|date()'); ?> }}</td>
```

Comment:
For printing out the fields, we should re-use the logic from here: https://github.com/sensiolabs/SensioGeneratorBundle/blob/master/Resources/skeleton/crud/views/index.html.twig.twig#L25

Honestly, that code is *ugly*! It would be ok to create a new PHP class to do a lot of this work, then just use that class here - e.g. 

```php
{{ <?= $helper->getPrintCode($entity_var_singular, $field['fieldName'], $field['type']); ?> }}
```

... which would print:

```twig
{{ product.createdAt ? product.createdAt|date('Y-m-d') : '' }}
```